### ISSUE TITLE
I've observed a difference in my results. Could you please help me understand the reason behind this discrepancy

### ISSUE THREAD

**Issue by WangYJ-WYJ** [NeuralCollapseApplications/FSCIL#7]
[20231213_170107.log](https://github.com/NeuralCollapseApplications/FSCIL/files/13658630/20231213_170107.log)

**Comment by HarborYuan** [comment]
Hi @WangYJ-WYJ ,

Thanks for your interest in our work.

When using different versions of PyTorch or different models of GPU, the results might be different due to the difference in the underlying implementation. You can consider adjusting hyperparameters such as the learning rate in your environment to further test the results.

You can also try other datasets first to troubleshoot the dataset issues.

For GPU and environment, you can consider referring to the [log](https://github.com/NeuralCollapseApplications/FSCIL/blob/main/logs/cub_inc.log), which is obtained under the current code.

Please let me know if you have any other question.

**Comment by WangYJ-WYJ** [comment]
> Hi @WangYJ-WYJ ,
> 
> Thanks for your interest in our work.
> 
> When using different versions of PyTorch or different models of GPU, the results might be different due to the difference in the underlying implementation. You can consider adjusting hyperparameters such as the learning rate in your environment to further test the results.
> 
> You can also try other datasets first to troubleshoot the dataset issues.
> 
> For GPU and environment, you can consider referring to the [log](https://github.com/NeuralCollapseApplications/FSCIL/blob/main/logs/cub_inc.log), which is obtained under the current code.
> 
> Please let me know if you have any other question.

config is the configuration file. work_dir is the folder path to save logs and weights. load_from is the path to obtain initialization weights during the incremental stage, and it should be consistent with work_dir.
